isso/ext/notifications.py

```diff
@@ -104,9 +109,10 @@ def __iter__(self):
         yield "comments.new:after-save", self.notify_new
         yield "comments.activate", self.notify_activated
 
-    def format(self, thread, comment, parent_comment, recipient=None, admin=False):
+    def format(self, thread, comment, parent_comment, recipient=None):
```

Comment:
Why is this changing the signature of format? It should only be touching upon the subject.